22:52:21 - ERR - getRecords - (OperationalError('could not translate host name "db_siug" to address: Host desconocido. \n'),)
22:52:21 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:53:20 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
22:53:20 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:55:15 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
22:55:15 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:55:39 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabasd\xbb\n', 85, 86, 'invalid continuation byte'),)
22:55:39 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:56:14 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabasd\xbb\n', 85, 86, 'invalid continuation byte'),)
22:56:14 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:57:14 - ERR - post - (<BadRequest '400: Bad Request'>,)
22:57:44 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
22:57:44 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
22:57:55 - ERR - post - (<BadRequest '400: Bad Request'>,)
22:58:15 - ERR - post - ("Error al Validar el Request -> {'usuario': ['Missing data for required field.'], 'contrasena': ['Missing data for required field.']}",)
22:58:40 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
22:58:40 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:01:13 - ERR - getRecords - (OperationalError('could not translate host name "dawa" to address: Host desconocido. \n'),)
23:01:13 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:01:47 - ERR - getRecords - (OperationalError('could not translate host name "postgres" to address: Host desconocido. \n'),)
23:01:47 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:04:09 - ERR - getRecords - (ValueError("invalid literal for int() with base 10: 'localhost'"),)
23:04:09 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:07:36 - ERR - getRecords - (OperationalError('could not translate host name "flask_db" to address: Host desconocido. \n'),)
23:07:36 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:09:10 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
23:09:10 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:09:13 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
23:09:13 - ERR - Login - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
23:12:16 - ERR - getRecords - (AttributeError("type object 'Parametros' has no attribute 'db_schema'"),)
23:12:16 - ERR - Login - ('Error al Buscar login de usuario',)
23:12:18 - ERR - getRecords - (AttributeError("type object 'Parametros' has no attribute 'db_schema'"),)
23:12:18 - ERR - Login - ('Error al Buscar login de usuario',)
23:14:25 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
23:14:25 - ERR - Login - ('Error al Buscar login de usuario',)
23:15:06 - ERR - getRecords - (AttributeError("type object 'Parametros' has no attribute 'db_schema'"),)
23:15:06 - ERR - Login - ('Error al Buscar login de usuario',)
23:18:15 - ERR - getRecords - (AttributeError("type object 'Parametros' has no attribute 'db_schema'"),)
23:18:15 - ERR - Login - ('Error al Buscar login de usuario',)
23:18:30 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabdawa\xbb\n', 85, 86, 'invalid continuation byte'),)
23:18:30 - ERR - Login - ('Error al Buscar login de usuario',)
23:18:58 - ERR - getRecords - (OperationalError('could not translate host name "b\'localhost\'" to address: Host desconocido. \n'),)
23:18:58 - ERR - Login - ('Error al Buscar login de usuario',)
23:19:02 - ERR - getRecords - (OperationalError('could not translate host name "b\'localhost\'" to address: Host desconocido. \n'),)
23:19:02 - ERR - Login - ('Error al Buscar login de usuario',)
23:19:24 - ERR - getRecords - (OperationalError('could not translate host name "b\'postgres\'" to address: Host desconocido. \n'),)
23:19:24 - ERR - Login - ('Error al Buscar login de usuario',)
23:20:18 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:20:18 - ERR - Login - ('Error al Buscar login de usuario',)
23:22:46 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:22:46 - ERR - Login - ('Error al Buscar login de usuario',)
23:22:56 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:22:56 - ERR - Login - ('Error al Buscar login de usuario',)
23:23:14 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:23:14 - ERR - Login - ('Error al Buscar login de usuario',)
23:25:55 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:25:55 - ERR - Login - ('Error al Buscar login de usuario',)
23:25:57 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:25:57 - ERR - Login - ('Error al Buscar login de usuario',)
23:25:58 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:25:58 - ERR - Login - ('Error al Buscar login de usuario',)
23:25:58 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "localhost" (::1), port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 85, 86, 'invalid continuation byte'),)
23:25:58 - ERR - Login - ('Error al Buscar login de usuario',)
23:30:13 - ERR - getRecords - (OperationalError('connection to server at "12.0.0.1", port 5432 failed: Connection timed out (0x0000274C/10060)\n\tIs the server running on that host and accepting TCP/IP connections?\n'),)
23:30:13 - ERR - Login - ('Error al Buscar login de usuario',)
23:30:41 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "127.0.0.1", port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 79, 80, 'invalid continuation byte'),)
23:30:41 - ERR - Login - ('Error al Buscar login de usuario',)
23:30:47 - ERR - getRecords - (UnicodeDecodeError('utf-8', b'connection to server at "127.0.0.1", port 5432 failed: FATAL:  la autentificaci\xf3n password fall\xf3 para el usuario \xabb\'dawa\'\xbb\n', 79, 80, 'invalid continuation byte'),)
23:30:47 - ERR - Login - ('Error al Buscar login de usuario',)
23:46:14 - ERR - getRecords - (OperationalError('connection to server at "127.0.0.1", port 5433 failed: FATAL:  password authentication failed for user "b\'dawa\'"\n'),)
23:46:14 - ERR - Login - ('Error al Buscar login de usuario',)
23:46:59 - ERR - getRecords - (OperationalError('connection to server at "localhost" (::1), port 5433 failed: FATAL:  password authentication failed for user "b\'dawa\'"\n'),)
23:46:59 - ERR - Login - ('Error al Buscar login de usuario',)
23:47:57 - ERR - getRecords - (OperationalError('connection to server at "localhost" (::1), port 5433 failed: FATAL:  password authentication failed for user "b\'dawa\'"\n'),)
23:47:57 - ERR - Login - ('Error al Buscar login de usuario',)
23:48:27 - ERR - getRecords - (OperationalError('connection to server at "localhost" (::1), port 5433 failed: FATAL:  password authentication failed for user "b\'dawa\'"\n'),)
23:48:27 - ERR - Login - ('Error al Buscar login de usuario',)
